src/OpenTelemetry.Instrumentation.Wcf/README.md

```diff
@@ -50,8 +49,8 @@ using var openTelemetry = Sdk.CreateTracerProviderBuilder()
 
 ## WCF Client Configuration (.NET Framework)
 
-Add the `IClientMessageInspector` instrumentation via a behavior extension on
-the clients you want to instrument:
+Configure the `TelemetryEndpointBehaviorExtensionElement` on the clients
```

Comment:
@repl-chris Do users need to do anything to their existing code to migrate to the new style when upgrading the package?

Reply:
No, there is no change needed to existing code to migrate to the new style - all the configuration stays the same.  I changed this README because it's not just a clientmessageinspector anymore, it is also a bindingelement.  But really, both of those details are irrelevant to someone using this library - as far as a user is concerned it's still just an endpointbehavior extension (just as it was before).

Reply:
Thanks for confirming!